packages/geoprocessing/src/aws/PreprocessingHandler.test.ts

```diff
@@ -46,6 +64,21 @@ test("Returns successful output as geojson", async () => {
   expect(body.data.type).toBe("Feature");
 });
 
+test("Preprocessor can accept extraParams", async () => {
+  const extraParams = { geography: "nearshore" };
+  const results = await extraParamHandler.lambdaHandler(
+    {
+      body: JSON.stringify({ feature, extraParams }),
+      headers: {},
+    } as APIGatewayProxyEvent,
+    {} as Context
+  );
```

Comment:
The handler that geoprocessing functions get wrapped in need to be called with an event payload, which includes headers and a `body` -- https://docs.aws.amazon.com/lambda/latest/dg/services-apigateway.html#apigateway-example-event.

Because the stack uses a `LambdaIntegration` the rest API will automatically pass any JSON sent in the body to the lambda handler (GeoprocessingHandler.ts in this case).